packages/headless/tsconfig.json

```diff
@@ -10,6 +10,9 @@
     "sourceMap": true,
     "outDir": "./dist",
     "baseUrl": "./",
+    "paths": {
+			"@modules/*": ["./src/modules/*"]
+		},
```

Comment:
dont change config files without our concerns. let us know if anything should be changed

Reply:
reverted